From: uazo <uazo@users.noreply.github.com>
Date: Sat, 28 Jun 2025 08:16:28 +0000
Subject: Disable css preferred text scale

Do not expose the user's preferred font scale to CSS.

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../blink/common/features_cc/Disable-CSSPreferredTextScale.inc | 1 +
 .../blink/renderer/platform/runtime_enabled_features.json5     | 3 +--
 2 files changed, 2 insertions(+), 2 deletions(-)
 create mode 100644 cromite_flags/third_party/blink/common/features_cc/Disable-CSSPreferredTextScale.inc

diff --git a/cromite_flags/third_party/blink/common/features_cc/Disable-CSSPreferredTextScale.inc b/cromite_flags/third_party/blink/common/features_cc/Disable-CSSPreferredTextScale.inc
new file mode 100644
--- /dev/null
+++ b/cromite_flags/third_party/blink/common/features_cc/Disable-CSSPreferredTextScale.inc
@@ -0,0 +1 @@
+SET_CROMITE_FEATURE_DISABLED(kCSSPreferredTextScale);
diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -1376,8 +1376,7 @@
       status: "test",
     },
     {
-      name: "CSSPreferredTextScale",
-      status: "stable",
+      name: "CSSPreferredTextScale", status: "test", public: true
     },
     // https://drafts.csswg.org/css-values-5/#progress
     // progress()
--
